spec/views/catalog/_index_list_default.html.erb_spec.rb

```diff
@@ -35,5 +37,9 @@
     expect(rendered).to include 'Test depositor_tesim'
     expect(rendered).to include '<span class="attribute-label h4">Rights:</span>'
     expect(rendered).to include 'Test rights_tesim'
+    expect(rendered).to include '<span class="attribute-label h4">Embargo release date:</span>'
+    expect(rendered).to include 'Test embargo_release_date_dtsi'
```

Comment:
if we can tweak this test to test (or include another) to ensure that dates come through human-readable, that would be ideal.